src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -39,6 +39,7 @@
         .protoVersionBound(packageConfig.protoVersionBound(language))
         .commonProtosVersionBound(packageConfig.commonProtosVersionBound(language))
         .authVersionBound(packageConfig.authVersionBound(language))
+        .packageName(packageConfig.packageName(language))
```

Comment:
I'm concerned about that. It would be easy to update to `protoPackageName`, but if `grpcPackageName` is being used by _anything else_ (note that this is _not_ a language-dependent transformer), then it will be a breaking change.